kie-remote/kie-remote-services/src/main/java/org/kie/remote/services/rest/jaxb/DynamicJaxbContext.java

```diff
@@ -86,27 +70,22 @@
 
  * <b>In all cases, please preserve existing tests when modifying this class!</b>
  */
-@ApplicationScoped
+
 public class DynamicJaxbContext extends JAXBContext {
 
     private static final Logger logger = LoggerFactory.getLogger(DynamicJaxbContext.class);
 
     // The contextsCache needs to be a ConcurrentHashMap because parallel operations (involing *different* deployments) can happen on it.
-    private static ConcurrentHashMap<String, JAXBContext> contextsCache = new ConcurrentHashMap<String, JAXBContext>();
+    static ConcurrentHashMap<String, JAXBContext> contextsCache = new ConcurrentHashMap<String, JAXBContext>();
```

Comment:
maybe better to put this contextsCache in manager and then let this class access manager instead? That way would could avoid static map in this class and make it close to be "pure" JAXBContext impl/extension.